app/assets/stylesheets/workarea/storefront/components/_haven_content_search_autocomplete.scss

```diff
@@ -6,9 +6,14 @@
 $search-autocomplete-content-link-color: $white !default;
 
 .search-autocomplete__content {
-    @extend %list-reset;
+    @include respond-to($medium-breakpoint) {
```

Comment:
I moved these styles over to an append rather than a full on override because a lot of this is already being done out of box for you, and the out-of-box CSS could get rendered accidentally if the load order in Gemfile is changed. This should ensure that your CSS isn't dependent on the gems load order.